media-sound/helvum/helvum-9999.ebuild

```diff
@@ -0,0 +1,165 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CRATES="
```

Comment:
..or actually doesn't need to be empty, you can remove CRATES entirely because of:
```
        if [[ -z ${CRATES} && "${PV}" != *9999* ]]; then
            eerror "undefined CRATES variable in non-live EAPI=8 ebuild"
            die "CRATES variable not defined"
        fi
```